pkgs/by-name/si/sirula/package.nix

```
@@ -1,36 +1,27 @@
-{
-  lib,
-  fetchFromGitHub,
-  rustPlatform,
-  pkg-config,
-  gtk3,
-  gtk-layer-shell,
-}:
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, gtk3, gtk-layer-shell, }:
 
 rustPlatform.buildRustPackage rec {
   pname = "sirula";
-  version = "1.0.0-unstable-2023-09-02";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "DorianRudolph";
     repo = pname;
```

Use of pname as a binding here is discouraged.
```suggestion
    repo = "sirula";
```